<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Align flanges</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Documenti/flamingo/icons/flangeAlign.svg</normaloff>../../../../Documenti/flamingo/icons/flangeAlign.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>&lt; select a target face &gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="btn1">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>(S)elect face</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnXY">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>XY</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnXZ">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>XZ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnYZ">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>YZ</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="X">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>normX</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="Y">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>normY</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="Z">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>normZ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="btnNorm">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Set notmal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn1</tabstop>
  <tabstop>btnXY</tabstop>
  <tabstop>btnXZ</tabstop>
  <tabstop>btnYZ</tabstop>
  <tabstop>X</tabstop>
  <tabstop>Y</tabstop>
  <tabstop>Z</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
